docs/development/testing.md

```diff
@@ -35,13 +35,22 @@ Integration tests are tests that use a k8s-conformance-compliant cluster and opt
 The tests are by default executed on every head-update of the main git branch and in case of a new release.
 In both cases the [script](../../.ci/integration-test) is executed, which 
 
-- creates a Gardener shoot cluster in the project *laas* on the Gardener Canary landscape. All such clusters have a name with the prefix *it-*.
+- creates a Gardener shoot cluster in the project *laas* on the Gardener Canary landscape
```

Comment:
I know only the `laas` project on the live landscape.

Reply:
I added you to the project (https://dashboard.garden.canary.k8s.ondemand.com/namespace/garden-laas/shoots). This is not our productive landscape.